2021/06/04 10:30:39 [error] 32#32: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://www.sym/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.23.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.23.0.3:9000", host: "www.sym"
2021/06/04 15:19:06 [emerg] 1#1: host not found in upstream "php-fpm:9000" in /etc/nginx/conf.d/default.conf:2
2021/06/05 08:55:51 [emerg] 1#1: host not found in upstream "php-fpm:9000" in /etc/nginx/conf.d/default.conf:2
2021/06/05 09:14:32 [error] 33#33: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "www.sym"
2021/06/05 09:14:41 [error] 33#33: *2 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://www.sym/favicon.ico" (from "http://www.sym/")" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.19.0.1, server: www.sym, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "www.sym", referrer: "http://www.sym/"
2021/06/05 09:14:46 [error] 33#33: *2 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://www.sym/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.19.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "www.sym"
2021/06/05 09:16:37 [error] 33#33: *6 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://www.sym/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.19.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "www.sym"
2021/06/05 09:16:41 [error] 33#33: *6 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://www.sym/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.19.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "www.sym"
2021/06/05 09:16:43 [error] 33#33: *6 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://www.sym/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.19.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.3:9000", host: "www.sym"
2021/06/05 09:38:21 [error] 34#34: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost"
2021/06/05 09:38:22 [error] 34#34: *1 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost/favicon.ico" (from "http://localhost/")" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "localhost", referrer: "http://localhost/"
2021/06/05 09:38:50 [error] 31#31: *6 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://www.sym/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 09:58:49 [error] 31#31: *9 FastCGI sent in stderr: "PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 10:03:25 [error] 31#31: *11 FastCGI sent in stderr: "PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 10:03:56 [error] 31#31: *11 FastCGI sent in stderr: "PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 10:05:03 [error] 31#31: *15 FastCGI sent in stderr: "PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 10:05:39 [error] 31#31: *15 FastCGI sent in stderr: "PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 11:05:52 [error] 31#31: *19 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/_intellij_phpdebug_validator.php"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2021/06/05 11:08:27 [error] 31#31: *21 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/_intellij_phpdebug_validator.php"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /_intellij_phpdebug_validator.php HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2021/06/05 12:07:01 [error] 34#34: *28 FastCGI sent in stderr: "PHP message: [info] Matched route "index".
PHP message: [critical] Uncaught PHP Exception ErrorException: "Warning: file_put_contents(data.json): Failed to open stream: Permission denied" at /var/www/src/Controller/DefaultController.php line 16" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 12:14:15 [error] 31#31: *31 FastCGI sent in stderr: "PHP message: [info] Matched route "index".
PHP message: [critical] Uncaught PHP Exception ErrorException: "Warning: file_put_contents(data.json): Failed to open stream: Permission denied" at /var/www/src/Controller/DefaultController.php line 16" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 14:05:48 [error] 31#31: *33 FastCGI sent in stderr: "PHP message: [info] Matched route "index".
PHP message: [critical] Uncaught PHP Exception ErrorException: "Warning: file_put_contents(data.json): Failed to open stream: Permission denied" at /var/www/src/Controller/DefaultController.php line 16" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:01:54 [error] 31#31: *36 FastCGI sent in stderr: "PHP message: [info] Matched route "index".
PHP message: [critical] Uncaught PHP Exception ErrorException: "Warning: file_put_contents(data.json): Failed to open stream: Permission denied" at /var/www/src/Controller/DefaultController.php line 16" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:02:28 [error] 31#31: *36 FastCGI sent in stderr: "PHP message: [info] Matched route "index".
PHP message: [critical] Uncaught PHP Exception ErrorException: "Warning: file_put_contents(data.json): Failed to open stream: Permission denied" at /var/www/src/Controller/DefaultController.php line 16" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:21:32 [error] 31#31: *45 FastCGI sent in stderr: "PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:27:25 [error] 31#31: *47 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:27:25 [error] 31#31: *47 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:28:06 [error] 31#31: *47 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:28:06 [error] 31#31: *47 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:45:51 [error] 31#31: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:45:51 [error] 31#31: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading upstream, client: 172.20.0.1, server: www.sym, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "www.sym"
2021/06/05 18:49:07 [error] 31#31: *54 FastCGI sent in stderr: "PHP message: [info] Matched route "index"" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1"
2021/06/05 18:49:08 [error] 31#31: *54 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://127.0.0.1/favicon.ico" (from "http://127.0.0.1/")" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.20.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://172.20.0.3:9000", host: "127.0.0.1", referrer: "http://127.0.0.1/"
2021/06/07 14:26:54 [error] 32#32: *5 FastCGI sent in stderr: "PHP message: [error] Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET http://localhost/"" at /var/www/vendor/symfony/http-kernel/EventListener/RouterListener.php line 136" while reading response header from upstream, client: 172.19.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.19.0.2:9000", host: "localhost"
